src/data.js

```diff
@@ -329,9 +333,12 @@ Artifact.prototype.remove = function(ignoreError) {
           this.details.container, this.details.path);
     // Validate that this is the configured container
     if (this.details.container !== this.blobStore.container) {
-      debug("[alert-operator] Expiring artifact with container: %s, which " +
-            "configured for use. Please investigate taskId: %s, runId: %s",
-            this.details.container, this.taskId, this.runId);
+      let err = new Error("Expiring artifact with container which isn't " +
+                          "configured for use. Please investigate!");
+      err.container = this.details.container;
+      err.taskId    = this.taskId;
+      err.runId     = this.runId;
+      this.monitor.reportError(err);
```

Comment:
ok, so I think that this should be used with "await" but I'll refrain from commenting on it anymore just to reduce the noise.